test/abilities/neutralizing-gas.test.ts

```diff
@@ -0,0 +1,197 @@
+import { AbAttrFlag } from "#enums/ab-attr-flag";
+import { AbilityId } from "#enums/ability-id";
+import { BattleCommand } from "#enums/battle-command";
+import { BattlerIndex } from "#enums/battler-index";
+import { MoveId } from "#enums/move-id";
+import { PhaseId } from "#enums/phase-id";
+import { PokeballType } from "#enums/pokeball-type";
+import { SpeciesId } from "#enums/species-id";
+import { Stat } from "#enums/stat";
+import type { CommandPhase } from "#phases/command-phase";
+import { GameManager } from "#test/test-utils/game-manager";
+import Phaser from "phaser";
+import { afterEach, beforeAll, beforeEach, describe, expect, it, vi } from "vitest";
+
+// TODO: https://github.com/pagefaultgames/pokerogue/pull/5381
+describe.todo("Abilities - Neutralizing Gas", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+    game.override
+      .moveset([MoveId.SPLASH])
+      .ability(AbilityId.NEUTRALIZING_GAS)
+      .battleType("single")
+      .disableCrits()
+      .enemySpecies(SpeciesId.MAGIKARP)
+      .enemyAbility(AbilityId.BALL_FETCH)
+      .enemyMoveset(MoveId.SPLASH);
+  });
+
+  it("should prevent other abilities from activating", async () => {
+    game.override.enemyAbility(AbilityId.INTIMIDATE);
+    await game.classicMode.startBattle(SpeciesId.FEEBAS);
+
+    game.move.select(MoveId.SPLASH);
+    await game.toEndOfTurn();
+
+    // Intimidate is suppressed, so the attack stat should not be lowered
+    expect(game.field.getPlayerPokemon().getStatStage(Stat.ATK)).toBe(0);
+  });
+
+  it("should allow the user's passive to activate", async () => {
+    game.override.passiveAbility(AbilityId.INTREPID_SWORD);
+    await game.classicMode.startBattle(SpeciesId.FEEBAS);
+
+    game.move.select(MoveId.SPLASH);
+    await game.toEndOfTurn();
+
+    expect(game.field.getPlayerPokemon().getStatStage(Stat.ATK)).toBe(1);
+  });
+
+  // TODO: https://github.com/Despair-Games/poketernity/issues/1211
+  it.todo("should activate before other abilities", async () => {
+    game.override.enemySpecies(SpeciesId.ACCELGOR).enemyLevel(100).enemyAbility(AbilityId.INTIMIDATE);
+
+    await game.classicMode.startBattle(SpeciesId.FEEBAS);
+
+    game.move.select(MoveId.SPLASH);
+    await game.toEndOfTurn();
+
+    // Intimidate is suppressed even when the user's speed is lower
+    expect(game.field.getPlayerPokemon().getStatStage(Stat.ATK)).toBe(0);
```

Comment:
There is a `toHaveStatStage` custom matcher that improves readability 

```suggestion
    expect(game.field.getPlayerPokemon()).toHaveStatStage(Stat.ATK, 0);
```